exercises/concept/regular-chatbot/.docs/instructions.md

```diff
@@ -0,0 +1,90 @@
+# Instructions
+
+## Introduction
+
+<aside>
+You have been hired as a Regular Expression Specialist in a company that is developing a Chatbot. 
+<br /><br />
+It is in a very basic phase of development, hence your mission is to use Regular Expressions to improve the chatbot's ability to understand and generate natural language.
+
+</aside>
+
+## Chatbot Command
+
+Apart from being smart, the Chatbot is also a loyal assistant.
+
+To ask something to the chatbot, the user must say the word “**Chatbot**” in the first position of the command. The software is still able to detect if it is being written in UPPERCASE or lowercase.
```

Comment:
I am not sure what you are trying to say with the second sentence here. It seems the function below does not care about upper- or lowercase but the sentence seems to suggest otherwise.

Reply:
In the moment of writing, I was also doubting if it was going to be easy to understand. You're right. 

I will try to re-frame it, so it is easily readable. Thanks!